client/src/components/Collections/CollectionCreatorIndex.vue

```diff
@@ -231,6 +234,13 @@ function redrawCreator() {
     }
 }
 
+function asSampleSheetType(): SampleSheetCollectionType {
```

Comment:
why not make that a `sampleSheetCollectionType` computed and return null if `collectionType` is not sample sheet ? That's reactive (I don't think the function call is) and you can just do `v-else-if="sampleSheetCollectionType"` below

Reply:
Yeah - this was to workaround TypeScript not realizing the if check mean this was of the proper type and not allowing me to use an ``as`` statement in attribute specification. I think the computed would address both of these more cleanly - I think unfortunately I have used this paradigm to work around type issues a lot in existing, committed code. I'll try to remember to swap to this style going forward if it works out.